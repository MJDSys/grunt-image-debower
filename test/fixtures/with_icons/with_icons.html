<ul>
<li>Test page</li>
<li>Icon old: <md-icon icon="/bower_components/icon-package/images/icon.svg"></md-icon></li>
<li>Icon new: <md-icon md-svg-src="/bower_components/icon-package/images/icon.svg"></md-icon></li>
<li>Object: <object class="some-class" data="/bower_components/icon-package/images/icon.svg"></object></li>
<li>Image: <img src="/bower_components/icon-package/images/icon.png"></li>
</ul>
