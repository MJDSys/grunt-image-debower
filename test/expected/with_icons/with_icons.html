<ul>
<li>Test page</li>
<li>Icon old: <md-icon icon="/images/icon-package/icon.svg"></md-icon></li>
<li>Icon new: <md-icon md-svg-src="/images/icon-package/icon.svg"></md-icon></li>
<li>Object: <object class="some-class" data="/images/icon-package/icon.svg"></object></li>
<li>Image: <img src="/images/icon-package/icon.png"></li>
</ul>
